### ISSUE TITLE
locale column in attribute export

### ISSUE THREAD

**Issue by akkermanronald** [akeneo-labs/EnhancedConnectorBundle#23]
Why are we getting locale column for fr_FR in the attribute export if French is disabled and not associated to the channel? 

<img width="203" alt="schermafdruk 2015-10-21 11 45 56" src="https://cloud.githubusercontent.com/assets/15016164/10633182/6a6ae7d2-77e9-11e5-9659-5b0160572a06.png">

**Comment by damien-carcel** [comment]
Hello.

Are you talking about the attribute export dedicated to PimGento or the standard product export?
Anyway, both exports use the same reader, directly provided by the PIM. So this could be not a EnhancedConnectorBundle problem, but a PIM one.

I recommand that you ask for help on our forums for this matter.

Regards,
Damien

**Comment by damien-carcel** [comment]
Hello again.

I dig a little more about it. Indeed, the problem is specific to the connector and to the PimGento attribute export. We will try to fix it as soon as possible.

Regards,
Damien

**Comment by akkermanronald** [comment]
Thanks, we look forward to it!

**Comment by damien-carcel** [comment]
Hello Ronald.

This is now fixed with the last 1.0.3 release of the connector.
Don't forget to clear your cache and to reload assets after update:

```
php app/console cache:clear --env=prod
php app/console pim:install:assets --env=prod
```

Regards,
Damien
